src/components/WidgetContainer/index.tsx

```diff
@@ -19,29 +19,30 @@ export default function WidgetContainer(props: {
 
   return (
     <React.StrictMode>
-      <Box
-        sx={{
-          px: 2,
-          py: 1,
-          m: 1,
-          borderRadius: 1,
-          bgcolor: "lightblue",
-          position: "relative",
-        }}
-      >
-        <Tooltip title={id} placement="left">
-          <InfoIcon
-            sx={{
-              position: "absolute",
-              top: 5,
-              right: 5,
-              zIndex: 2000,
-              fontSize: 15,
-            }}
-          />
-        </Tooltip>
-        {inner}
-      </Box>
+      <Draggable>
+        <Box
+          sx={{
+            px: 2,
+            py: 1,
+            m: 1,
+            borderRadius: 1,
+            // bgcolor: "lightblue",
```

Comment:
Is there a reason for removing this? If you think the lightblue background color is too distracting you may also try the following:

```suggestion
            border: "2px solid black",
            bgcolor: "rgba(0, 0, 0, 0.2)",
```